WooCommerce/Classes/ViewRelated/ReusableViews/ErrorSectionHeaderView.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+/// Displays error for the table view section.
+///
+final class ErrorSectionHeaderView: UITableViewHeaderFooterView {
+
+    @IBOutlet private weak var titleLabel: UILabel!
+
+    // MARK: - Overridden Methods
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        tintColor = .clear
+
```

Comment:
Is this a desired space?

Reply:
not intentional! removed in https://github.com/woocommerce/woocommerce-ios/pull/1695/commits/958af2770f9ae50478664afd9ac76da16cf17269